api/room/schema_query.py

```diff
@@ -167,15 +170,16 @@ def resolve_all_rooms(self, info, **kwargs):
 
     @Auth.user_roles('Admin')
     def resolve_all_remote_rooms(self, info):
-        service = Credentials().set_api_credentials()
         page_token = None
+        switcher = map_remote_room_location_to_filter()
```

Comment:
please use self-explanatory variables, switcher may not be that show what the variable really does

Reply:
I have implemented this @michael-basweti